agent/tool-scripts/prometheus-metrics

```diff
@@ -168,30 +164,40 @@ case "$mode" in
 	;;
 	start)
 	mkdir -p $tool_output_dir/json
-	while read -u 9 master; do
-		# fetch keys from openshift master and set default values for cert and key vars if not defined
+	while read -u 9 file; do
+		# read the input file for host, port, cert and key if any. In case the file doesn't have certs, port and they 
+		# are not defined while registering the tool, the default values are set.
+		# host cert=<cert> key=<key> port=<port>
+		host=$(echo $file | awk -F " " '{ print $1 }')
+		port=$(echo $file | grep -w "port" | awk -F " " '{ print $2 }' | awk -F "=" '{print $2}')
+		cert=$(echo $file | grep -w "cert" | awk -F " " '{ print $3 }' | awk -F "=" '{print $2}')
+		key=$(echo $file | grep -w "key" | awk -F " " '{ print $4 }' | awk -F "=" '{print $2}')
+		# fetch keys from openshift master and set default values for port, cert and key vars if not defined
```

Comment:
Settings defined from the command line will override the values defined in the file. I modified the script to do that.